torch/_inductor/codegen/cpp_wrapper_cpu.py

```diff
@@ -731,13 +731,19 @@ def codegen_model_constructor(self):
                 self.prefix.writeline(
                     f"constants_info_[{idx}].offset = {tensor.storage_offset()};"
                 )
+
+                # For data_size, we always align it to 64.
+                # When loading the constants, the valid data will depends on the size
+                # not the data_size so there won't be correctness issue.
+                from .memory_planning import _align
```

Comment:
ditto: perhaps better to move to a more common place?

Reply:
Moved `_align` to `torch/_inductor/utils.py`